Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py

```diff
@@ -225,32 +409,262 @@ def generate_dbotscore(response):
+        if dbot_score.score >= 2:
+            url_outputs['Malicious'] = {
+                'Vendor': 'ANYRUN',
+                'Description': threat_text
+            }
+        returned_data.append(CommandResults(
+            outputs_prefix='URL',
+            outputs_key_field=['Data'],
+            outputs=url_outputs,
+            indicator=Common.URL(
+                url=main_object.get('url'),
+                dbot_score=dbot_score,
+            )
+        ))
+
+    # Check if network information is available in the report
+    if 'network' in data:
+        network_data = data.get('network')
+
+        # Then add all the network-related indicators - 'connections'
+        if 'connections' in network_data:
+            connections = network_data.get('connections')
+            for current_connection in connections:
+                reputation = current_connection.get('Reputation')
+                if reputation in reputation_map.keys():
+                    current_dbot_score = Common.DBotScore(
+                        indicator=current_connection.get('IP'),
+                        indicator_type=DBotScoreType.IP,
+                        integration_name='ANYRUN',
+                        score=reputation_map[reputation]
+                    )
+                    relationships = [EntityRelationship(
+                        name=EntityRelationship.Relationships.COMMUNICATED_WITH,
+                        entity_a=main_entity,
+                        entity_a_type=main_entity_type,
+                        entity_b=current_connection.get('IP'),
+                        entity_b_type=FeedIndicatorType.IP,
+                        brand="ANYRUN"
+                    )]
+                    ip_indicator = Common.IP(
+                        ip=current_connection.get('IP'),
+                        asn=current_connection.get('ASN'),
+                        port=current_connection.get('Port'),
+                        geo_country=current_connection.get('Country'),
+                        dbot_score=current_dbot_score,
+                        relationships=relationships
+                    )
+                    if current_connection.get('IP') not in [
+                        x.indicator.ip for x in returned_data if isinstance(x.indicator, Common.IP)
+                    ]:
+                        returned_data.append(CommandResults(
+                            readable_output=tableToMarkdown(
+                                f"{current_connection.get('IP')}",
+                                [{
+                                    "Description": f"This IP was observed after detonation of {main_entity} in ANYRUN"
+                                }]
+                            ),
+                            indicator=ip_indicator,
+                            relationships=relationships
+                        ))
+
+        # Then add all the network-related indicators - 'dnsRequests'
+        if 'dnsRequests' in network_data:
```

Comment:
Same notes as above section (the connections section) ⬆️

Reply:
Agreed